__tests__/endpoints.test.js

```diff
@@ -90,3 +119,47 @@ describe('GET /api/articles/:article_id', () => {
       });
   });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+  test('STATUS: 200, Returns updated object in new state', () => {
```

Comment:
This test is looking good! I'd throw in another test just to ensure we can pass in a negative number to decrement the vote count

Reply:
Have added negative inc_votes test - passing